packages/dialtone-vue2/components/pagination/pagination_variants.story.vue

```diff
@@ -38,6 +38,20 @@
         :page-number-aria-label="getPageNumberAriaLabel"
       />
     </div>
+    <div class="d-m32">
+      <p class="d-my16 d-fs-200 d-fw-bold">
+        Hide edges
+      </p>
+      <dt-pagination
+        :total-pages="10"
+        :active-page="5"
+        :aria-label="'pagination with separator on both sides'"
+        :prev-aria-label="'previous'"
+        :next-aria-label="'next'"
```

Comment:
double quoting like this technically works but is kinda weird, just remove the `:`

```
aria-label="pagination with separator on both sides"
prev-aria-label="previous"
...
```